vnext/src/babel-plugin-codegen/index.js

```diff
@@ -201,7 +201,16 @@ module.exports = function({parse, types: t}) {
            * Insert the TurboModule schema into the TurboModuleRegistry.(get|getEnforcing)
            * call.
            */
-          if (this.turboModuleRequireCallExpressions.length > 0) {
+
+          // Disabling TurobModule processing for react-native-web NPM module
```

Comment:
Typo:
```suggestion
          // Disabling TurboModule processing for react-native-web NPM module
```

Reply:
This is direct from FB. We wouldn't want to diverge locally.

Reply:
k I'll send a PR.

Reply:
Submitted [PR #30723](https://github.com/facebook/react-native/pull/30723) to [facebook/react-native](https://github.com/facebook/react-native)